src/_includes/components/navigation.html

```diff
@@ -1,15 +1,15 @@
 <nav class="site-nav" aria-label="Main">
     <div class="flexer">
         <a href="{{ navigation.donate.url }}" class="c-btn c-btn--primary donate-link">{{navigation.donate.text}}</a>
-        <button class="site-nav-toggle" aria-label="Menu" id="nav-toggle" hidden>
+        <button class="site-nav-toggle" aria-label="Menu" id="nav-toggle">
             <svg width="20" height="20" viewBox="20 20 60 60">
                 <path id="ham-top"    d="M30,37 L70,37 Z" stroke="currentColor"></path>
                 <path id="ham-middle" d="M30,50 L70,50 Z" stroke="currentColor"></path>
                 <path id="ham-bottom" d="M30,63 L70,63 Z" stroke="currentColor"></path>
             </svg>
         </button>
     </div>
-    <div class="nav-panel" data-open="false" id="nav-panel">
```

Comment:
Removing `data-open="false" ` alone **fixes the issue** but if we do this change alone then we will experience **CLS in mobile view**.

### Removing `data-open="false" ` alone :

 https://user-images.githubusercontent.com/30730124/172021784-7f5e7cb2-91e4-4b11-ae05-e750f3c8ff88.mov  

### After adding [<style>](https://github.com/eslint/playground/pull/64/files/efd44450852b8a85c918886ed6e405cbee84054c#r887329247) tag :

  https://user-images.githubusercontent.com/30730124/172021826-f43157be-8339-44d1-8008-634601f89ab9.mov